### PR TITLE
Web UI dependency updates

### PR DESCRIPTION
**Pull request by james-jory**
*Description of changes:*

- Bump hightlight.js and immer.js versions due to vulnerability reports.
- Local web-ui testing fix to set default Segment write key to `NONE` in .env file which is expected in the code and index.html for not-configured state.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/web-ui/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@optimizely/optimizely-sdk": "4.0.0",
     "amplitude-js": "^5.8.0",
-    "aws-amplify": "^3.0.23",
+    "aws-amplify": "^3.0.24",
```
Comment (kazbaig) [inline src/web-ui/package.json:14]: This on its own actually resolves all the dependency issues according `npm audit`. This is the case even with all of the dependencies removed.

File: src/web-ui/package.json [hunk 2]
```diff
@@ -41,6 +41,8 @@
   "resolutions": {
     "acorn": "^6.4.1",
     "elliptic": "^6.5.3",
+    "highlight.js": ">=10.4.1",
```
Comment (BastLeblanc) [inline src/web-ui/package.json:44]: do you need to force these ? is it not induced by amplify?
as suggested I would reduce to minimum versioning

Comment (james-jory) [reply src/web-ui/package.json:44]: Yes, for highlight.js. The immer issue can be resolved by upgrading aws-amplify.

**Comment by kazbaig** [comment]
Not seeing any vulnerabilities from `npm audit` after installing but I do see some changes in `package-lock.json` that need to be committed.

Also, I fresh installed my packages after removing all the packages in the `resolutions` field and still saw 0 vulnerabilities from `npm audit` so I still think the resolutions are not needed.

**Pull request closed without merging**